src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAsyncIteratorTests.cs

```diff
@@ -133,7 +133,7 @@ private static void VerifyMissingType(string source, WellKnownType type, params
         // Instrumentation to investigate CI failure: https://github.com/dotnet/roslyn/issues/34207
         private CSharpCompilation CreateCompilationWithAsyncIterator(string source, CSharpCompilationOptions options = null, CSharpParseOptions parseOptions = null)
             => CreateCompilationWithTasksExtensions(new[] { (CSharpTestSource)CSharpTestBase.Parse(source, filename: "source", parseOptions), CSharpTestBase.Parse(AsyncStreamsTypes, filename: "AsyncStreamsTypes", parseOptions) },
-                options: options, parseOptions: parseOptions);
```

Comment:
>parseOptions: parseOptions [](http://example.com/codeflow?start=34&length=26)

Why drop `parseOptions`?

Reply:
Because the syntax trees already have parseOptions - there is an assert failing if the options are specified both in trees and explicitly for the compilation - it wasn't hit just because it wasn't called with non-null `parseOptions` until now.